backend/src/main/resources/graphql/main.graphqls

```diff
@@ -392,6 +394,8 @@ type Query {
     facilityId: ID
     includeArchived: Boolean = false
```

Comment:
⚠️ for backward compatibility -- will deprecate in separate PR once this one is deployed